### ISSUE TITLE
When downloading if replacing content with a smaller file it does not properly replace

### ISSUE THREAD

**Issue by Onoitsu2** [LiquidFenrir/MultiUpdater#50]
It will download the content, however will place it inside the boundaries of the existing file, leaving a garbage chunk at the end it seems. The only workaround I've found for this in current versions is to download the file again. 

For example using my .json from my server, I have to download it twice if I cut something out of it, to ensure I won't start up MultiUpdater and receive a parsing error.

**Comment by LiquidFenrir** [comment]
are you using the latest version?

**Comment by Onoitsu2** [comment]
Indeed, using the latest version. Have found a workaround, and that is to download the file twice in succession, and then restart the app to use the updated config.
